pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -573,7 +575,12 @@ func convertToPrincipal(subject *rbacproto.Subject) *policyproto.Principal {
 	}
 
 	if subject.Group != "" {
-		rbacLog.Errorf("ignored Subject.group %s, not implemented", subject.Group)
+		if subject.Properties["request.auth.claims[groups]"] == "" {
```

Comment:
Better log if `subject.Group != ""` and `subject.Properties["request.auth.claims[groups]"] != ""`, also should use `attrRequestClaims` and make the `groups` to `attrRequestClaimsGroups`.

Reply:
Done.